clients/drcachesim/common/options.cpp

```diff
@@ -42,6 +42,20 @@ droption_t<bool> op_offline
  "If this option is enabled, trace data is instead written to files in -outdir "
  "for later offline analysis.  No simulator is executed.");
 
+droption_t<unsigned int> op_num_threads
+(DROPTION_SCOPE_CLIENT, "num_threads", 0, "Number of sideline threads for writing traces",
+ "For the offline analysis mode (when -offline is requested), specifies the number "
+ "of sideline threads to be used to write trace files out.  "
+ "Only 0 and 1 is currently supported.  "
+ "0 means synchronized write without any sideline threads.  ");
```

Comment:
A threading pool with more than one thread has a complex problem, i.e, sideline thread vs open files.
The simplest way is one open file per sideline thread, which requires us to change raw2trace.
If we want to keep current one file per app-thread, writing to same file with multiple threads is complex.
I will go with the one file per sideline thread.